pkg/antctl/raw/check/cluster/test_checkCNIExistence.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2024 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"antrea.io/antrea/pkg/antctl/raw/check"
+)
+
+type checkCNIExistence struct{}
+
+func init() {
+	RegisterTest("Check if another CNI is Present", &checkCNIExistence{})
+}
+
+func (t *checkCNIExistence) Run(ctx context.Context, testContext *testContext) error {
+	pods, err := testContext.client.CoreV1().Pods(antreaNamespace).List(ctx, metav1.ListOptions{LabelSelector: "name=cluster-check"})
+	if err != nil {
+		return fmt.Errorf("failed to list Pods: %v", err)
+	}
+	testContext.Log("Checking CNI configurations in Pod: %s", pods.Items[0].Name)
+	command := []string{"ls", "/etc/cni/net.d"}
+	output, _, err := check.ExecInPod(ctx, testContext.client, testContext.config, antreaNamespace, pods.Items[0].Name, "", command)
```

Comment:
the file /etc/cni/net.d is usually accessible at node level and can be denied access at pod level ! does it succeed to list the antrea config file using ExexInPod using "ls /etc/cni/net.d" command

Reply:
Hi Rajnish, Glad to meet you :)
Yes, current configuration can access the given path and have been tested with Antrea and Calico files.